ansible/monitoring.yml

```diff
@@ -15,6 +15,55 @@
         tasks_from: config.yml
       tags: config
 
+    - name: Define tmp directories on tmpfs
+      blockinfile:
+        path: /etc/tmpfiles.d/podman.conf
+        create: yes
+        block: |
+          d {{ podman_tmp_dir_root }}/{{ item.name }}/libpod/tmp 0755 podman podman
+          Z {{ podman_tmp_dir_root }}/{{ item.name }}            0755 podman podman
+      become: yes
+      loop: "{{ appliances_local_users_podman }}"
+      register: podman_tmp_dirs
+
+    - name: Create tmp directories
+      command: systemd-tmpfiles --create
+      become: true
+      when: podman_tmp_dirs.results | selectattr('changed') | list | length > 0 # when: any changed
+
+    - name: Create podman configuration directories
+      file:
+        path: "{{ item.home }}/.config/containers/"
+        state: directory
+        owner: "{{ item.name }}"
+        group: "{{ item.name }}"
+      become: yes
+      loop: "{{ appliances_local_users_podman }}"
```

Comment:
The other possibility is to make this code into a utility role that the podman roles all use, passing the relevant user e.g opendistro_user, kibana_user, filebeat_user. That would make the roles more usable in isolation. I don't think this is critical mind...

Reply:
That would be possible. I felt like this is all a bit of a hack to work around systemd/rhel/podman limits/interactions so I'd hope it disappears entirely when either a) we have user services working or b) the podman patch to remove /tmp/containers-users-* files on reboot.